app/src/org/commcare/views/widgets/ImageWidget.java

```diff
@@ -91,7 +92,7 @@ public ImageWidget(final Context context, FormEntryPrompt prompt, PendingCallout
         mErrorTextView.setText("Selected file is not a valid image");
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/5a959df2d1681e0a640ebb72d4513eaecefa585e/app/src/org/commcare/views/widgets/ImageWidget.java#L92"></span>String literal in <code>setText</code> can not be translated. Use Android resources instead.</td>
    </tr>
  </tbody>
</table>